BEP1.md

```diff
@@ -1,51 +1,51 @@
 
-For Binance Chain contributors, BEPs are a convenient way to track the progress of their implementation. This will give end users a convenient way to know the current status of a given feature, function or improvement.
+For BNB Chain contributors, it is a convenient way to track the progress of their implementation by BEPs. It will help end users to know the status of a given feature, function or improvement.
 
 ##  3.  BEP Types
 
 There are three types of BEP:
 
-- Feature Request: A Feature Request BEP describes functional changes on Binance Chain or/and Binance DEX, such as a change to the network protocol, proposer selection mechanism in consensus algorithm, change in block size or fee mechanism in application level.
-- Improvement: Improvements are advice gathered from the community.
-- Standard: This kind of proposal will change the workflow of Binance Chain working process, like this BEP itself.
+- **Standards**: A Standards BEP describes functional changes on BNB Chain, such as a change to the network protocol, proposer selection mechanism in consensus algorithm, change in block size or fee mechanism in application level. It will effect the implementation of BNB Chain.
+- **Information**: An Information BEP will clarify some concepts of the BNB Chain, it may not effect the BNB Chain client implementation.
+- **Process**: This kind of proposal will change the workflow of BNB Chain working process, like this BEP itself.
 
 ## 4.  BEP Workflow
-
-![img](https://lh3.googleusercontent.com/auaU1Ur5SZZKVcwQrFrHN-3T1vUP8C1wCjFWqJzzJpcdVCHR4JXe1Mzm7hCFuoEqVjkUOZIF5WrwK47jGw8r3wp9RLEPvAWY6DOsncNz0yCeVc84qew1Xf7ouk1qrcNdXpjGSNQG)
+![overall workflow](./assets/bep-1/workflow.png)
 
 *Figure 1: BEP workflow*
 
-Each status change is requested by the BEP author and reviewed by the BEP editors. Use a pull request to update the status.
+Once the idea is accepted, each status change is requested by the BEP author and reviewed by the BEP editors. Use a pull request to update the status.
 
-- Work in progress (WIP) -- A community member will write a draft BEP as a pull request.
-- Draft -- An editor will add notes and some requests, then if all changes are made, it will go to the next stage. If it’s not approved, it will be dropped.
-- Final -- This BEP is ready to be implemented
-- Implementation -- the development team could start work on this BEP
+- **Idea**: If you have an idea but not sure if it worths a BEP or not, you may discuss with the community first before you put too much effort. You may post your idea in [bnb chain forum](https://forum.bnbchain.org/) and visit [our discord channel](https://discord.gg/bnbchain) to let the community know it.
+- **Pre-Draft**: If your idea is accepted, it is still not a BEP yet. A formal document will be needed to describe your idea, the Pre-Draft document should follow the BEP format. And you need to create a pull request and the BEP editors will review it.
```

Comment:
why do we need a `pre-draft` status, when the pr is raised, we can view it as a draft and we can  review and discuss about the proposed BEP

Reply:
pre-draft is not a BEP, it needs to be accepted and it will enter draft state with a BEP number